scipy/stats/_continuous_distns.py

```diff
@@ -4117,6 +4117,26 @@ def _sf(self, x, mu):
     def _cdf(self, x, mu):
         return np.exp(self._logcdf(x, mu))
 
+    def _ppf(self, x, mu):
+        with np.errstate(divide='ignore', over='ignore', invalid='ignore'):
```

Comment:
I'll have to put this on my list of un-handled Boost warnings to track down

Reply:
Yeah I think they only happen when it's evaluated in the wrong tail, in which case I re-evaluate in the correct tail, so I thought they were safe to ignore for now.